python/oneflow/framework/tensor.py

```diff
@@ -486,6 +486,12 @@ def _triu(self, diagonal=0):
 
 
 def _uniform(self, a=0, b=1):
+    if isinstance(a, Tensor):
+        assert a.ndim == 0 and a.nelement() == 1  , "a must be a integer or scalar tensor!"
```

Comment:
```suggestion
        assert a.ndim == 0 and a.nelement() == 1  , "a must be a number or scalar tensor!"
```

我的理解是 float 和 integer 其实都可以？错误提示不应该局限是整型？

Reply:
嗯嗯，是的